### PR TITLE
DSD-802: Chakra update

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Fixes JIRA ticket [DSD-802](https://jira.nypl.org/browse/DSD-802)

## This PR does the following:

- Updates Chakra to version 1.8.5 or higher and 1.8.0 or higher for peer dependency.
- Added the `@chakra-ui/storybook-addon` package that integrates chakra with storybook. This allows us to make a minor change by removing the `DSProvider` we added to the story wrapper. This still _only_ works for `<Story>` blocks and `<Canvas>` blocks still need to use the DSProvider.

## How has this been tested?

Locally in Storybook.

## Accessibility concerns or updates

- N/A

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] View [the example in Storybook]()

### CODE REVIEW

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr968-culk9qeq9ca23yb78lwf7yqr5lfh4i6m.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6262d05f2e91387b1723e9fe

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr968-culk9qeq9ca23yb78lwf7yqr5lfh4i6m.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6262d05f2e91387b1723e9fe

**Review by bigfishdesign13 — APPROVED** [review]
Looks good.

File: .storybook/preview.tsx [hunk 1]
```diff
@@ -4,17 +4,12 @@ import "!style-loader!css-loader!sass-loader!../src/styles.scss";
...
 import { addDecorator, addParameters } from "@storybook/react";
-import { DocsContainer, DocsPage } from "@storybook/addon-docs";
 import { withTests } from "@storybook/addon-jest";
 
-import DSProvider from "../src/theme/provider";
+import nyplTheme from "../src/theme";
 import results from "../.jest-test-results.json";
 
 addParameters({
-  docs: {
```
Comment (oliviawongnyc) [inline .storybook/preview.tsx:14]: Why is this no longer needed?

**Review by oliviawongnyc — APPROVED** [review]
Just added one question!

File: .storybook/preview.tsx (hunk 1 above)
Comment (EdwinGuzman) [reply .storybook/preview.tsx:14]: At some point, this setup was needed for the docs page. Newer versions of Storybook included the docs page automatically so it is safe to remove. I didn't know this but it probably wasn't needed for the past few versions. It'll be great to take a look at the Storybook configs and update them -- they're not necessarily old, but an update would be nice.

**Comment by EdwinGuzman** [comment]
There are some issues upgrading but it seems like `framer-motion` needed to be set to v4 instead of the latest (v6). https://github.com/storybookjs/storybook/issues/16690#issuecomment-1079736501

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr968-culk9qeq9ca23yb78lwf7yqr5lfh4i6m.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6262d05f2e91387b1723e9fe

**Pull request merged**